packages/flutter_tools/lib/src/commands/build_web.dart

```diff
@@ -102,12 +102,21 @@ class BuildWebCommand extends BuildSubCommand {
         help: 'Compile to WebAssembly rather than JavaScript.\nSee $kWasmPreviewUri for more information.',
         negatable: false,
       );
+      argParser.addFlag(
+        'omit-type-checks',
+        help: 'Omit type checks in Wasm output.',
+        negatable: false,
+      );
     } else {
-      // Add the flag as hidden. Will give a helpful error message in [runCommand] below.
+      // Add the flags as hidden. Will give a helpful error message in [runCommand] below.
       argParser.addFlag(
         FlutterOptions.kWebWasmFlag,
         hide: true,
```

Comment:
Rather than duplicating these in the else block, is it possible to just put `hide: featureFlags.isFlutterWebWasmEnabled` on each of the flags?

Reply:
yup